Tella/Scenes/Uwazi/ViewModel/UwaziTemplateViewModel.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  UwaziTemplateViewModel.swift
+//  Tella
+//
+//  Created by Gustavo on 31/07/2023.
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class UwaziTemplateViewModel: ObservableObject {
+    
+    var mainAppModel : MainAppModel
+    
+    @Published private var templates : [CollectedTemplate] = []
+    @Published private var downloadedTemplates : [CollectedTemplate] = []
```

Comment:
In my opinion it's better to create a view model for Adding a template and other for getting downloaded Template(main view), What do you think?

Reply:
I created a new `AddTemplateVM` and `DownloadedTemplatesVM` so we divide the responsibilities between both view models